### PR TITLE
init commit. get json and fill main order table

### PR DESCRIPTION
**Pull request by sergeyarhutik**
app gets order list and draws orders table

### CODE REVIEW

**Review by RomanukAlex — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
...
+Link for download with instructions: https://github.com/Java-Arctic-Ratel/repair-service
+
+## Download project from github URL:
+git clone https://github.com/Java-Arctic-Ratel/repair-service-web-swing.git
+
+## Go into the repository: 
+cd repair-service-web-swing
+
+## Build application from base directory
+mvn clean install
```
Comment (aleksieiev) [inline README.md]: 1st row: `Build`
2nd row: `install`
Misunderstanding from my side

File: src/main/java/com/epam/repair/model/Appearance.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "appearanceName"
+})
```
Comment (aleksieiev) [inline src/main/java/com/epam/repair/model/Appearance.java]: 1 field.
what for?

File: src/main/java/com/epam/repair/model/Address.java [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
+        "houseNumber",
+        "apartmentNumber",
+        "city",
+        "street"
+})
+
```
Comment (aleksieiev) [inline src/main/java/com/epam/repair/model/Address.java]: is it really important?

File: src/main/java/com/epam/repair/service/OrderService.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        ObjectMapper mapper = new ObjectMapper();
+        RepairOrderPage repairOrderPage = null;
+        try {
+            repairOrderPage = mapper.readValue(response.body(), RepairOrderPage.class);
+        } catch (JsonProcessingException e) {
+            System.out.println("Can not get parse response body");
+        }
+        assert repairOrderPage != null;
+        return repairOrderPage.content;
+    }
```
Comment (aleksieiev) [inline src/main/java/com/epam/repair/service/OrderService.java]: All antipatterns are here

File: src/main/java/com/epam/repair/ui/forms/order/model/OrderTableModel.java [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+            case 12:
+                return repairOrder.getDevice().getAppearance().getAppearanceName();
+            case 13:
+                return repairOrder.getRepairOrderCostEstimated();
+            case 14:
+                return repairOrder.getStatus().getStatusName();
+            default:
+                return "";
+        }
+    }
```
Comment (aleksieiev) [inline src/main/java/com/epam/repair/ui/forms/order/model/OrderTableModel.java]: Loooooooooong and ugly.

File: src/main/java/com/epam/repair/util/DateFormatter.java [hunk 6]
```diff
@@ -0,0 +1,13 @@
+package com.epam.repair.util;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DateFormatter {
+
+    public static String formatDate(Date date) {
+        SimpleDateFormat formatter = new SimpleDateFormat("dd-MM-yyyy");
```
Comment (aleksieiev) [inline src/main/java/com/epam/repair/util/DateFormatter.java]: new object every time. really?
Do you know that you have spring in this app?

**Review by aleksieiev — CHANGES_REQUESTED** [review]

**Pull request merged**